pkg/core/blockchain.go

```diff
@@ -883,13 +883,14 @@ func (bc *Blockchain) addHeaders(verify bool, headers ...*block.Header) error {
 			continue
 		}
 		bc.headerHashes = append(bc.headerHashes, h.Hash())
+		buf.WriteB(storage.ExecBlock)
```

Comment:
@roman-khimov, this change is backwards-incompatible, shouldn't we increase storage version?

Reply:
I've been waiting for this comment. Sure.

Reply:
Fixed.